### ISSUE TITLE
Train and validation loss are 0.0 when fitting on text only

### ISSUE THREAD

**Issue by madisonmay** [IndicoDataSolutions/finetune#83]

**Comment by benleetownsend** [comment]
When this is fixed, we need log entries to say whether its fitting just on text or supervised.

**Comment by xuy2** [comment]
Hi, has it been fixed? Does "fitting on text only" mean pre-training the language model?

**Comment by madisonmay** [comment]
Hi there @xuy2,

Your interpretation is correct -- unfortunately this is still an open issue.  Will hopefully get around to implementing a fix soon.

**Comment by madisonmay** [comment]
No longer logged via finetune 0.5.10 -- closing.
